gpu_multi_process_run.sh

```diff
@@ -32,10 +32,15 @@ set_nccl_gpudirect_tcpx_specific_configuration() {
     export TF_CPP_MAX_LOG_LEVEL=100
     export TF_CPP_VMODULE=profile_guided_latency_estimator=10
     export XLA_PYTHON_CLIENT_MEM_FRACTION=0.85
+    shopt -s globstar nullglob
+    IFS=:$IFS
+    set -- /usr/local/cuda-12.*/compat
+    export LD_LIBRARY_PATH="${1+:"$*"}:${LD_LIBRARY_PATH}:/usr/local/tcpx/lib64"
```

Comment:
We shouldn't take a dependency on the compat binaries long-term, these don't have LTS guarantees and can break with cuda version updates. 

Unfortunately JAX has pulled ahead to 12.6 and GKE has 12.4, so I don't see a way around this at the moment without changing our build infra to pin on 12.4.